commit b70098f9dba68c3255b2d45a319d886b7da9cf3b
Author: Stefano Babic <sbabic@denx.de>
Date:   Fri Jan 20 10:38:20 2017 -0500

    VME: restore bus_remove function causing incomplete module unload
    
    commit 9797484ba83d68f18fe1cbd964b7cd830f78f0f7 upstream.
    
    Commit 050c3d52cc7810d9d17b8cd231708609af6876ae ("vme: make core
    vme support explicitly non-modular") dropped the remove function
    because it appeared as if it was for removal of the bus, which is
    not supported.
    
    However, vme_bus_remove() is called when a VME device is removed
    from the bus and not when the bus is removed; as it calls the VME
    device driver's cleanup function.  Without this function, the
    remove() in the VME device driver is never called and VME device
    drivers cannot be reloaded again.
    
    Here we restore the remove function that was deleted in that
    commit, and the reference to the function in the bus structure.
    
    Fixes: 050c3d52cc78 ("vme: make core vme support explicitly non-modular")
    Cc: Manohar Vanga <manohar.vanga@gmail.com>
    Acked-by: Martyn Welch <martyn@welchs.me.uk>
    Cc: devel@driverdev.osuosl.org
    Signed-off-by: Stefano Babic <sbabic@denx.de>
    Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1330_b70098f9dba68c3255b2d45a319d886b7da9cf3b.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1330_b70098f9dba68c3255b2d45a319d886b7da9cf3b.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
